Hardware/Hardware/CardReader/StripeCardReader/PaymentIntent+Stripe.swift

```diff
@@ -32,6 +32,10 @@ protocol StripePaymentIntent {
     var charges: [StripeTerminal.Charge] { get }
 }
 
-
-extension StripeTerminal.PaymentIntent: StripePaymentIntent { }
 #endif
+
+extension StripeTerminal.PaymentIntent: StripePaymentIntent {
+    var id: String {
+        stripeId ?? ""
+    }
+}
```

Comment:
> SCPPaymentIntent.stripeId is now nullable to support offline payments.

Making `stripeID` nullable involves more changes that I think we can skip for the moment, since we do not support offline payments and will be non-null in all cases. Happy to iterate further if you believe is needed.